docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md

```diff
@@ -52,12 +52,19 @@ In this step, you will configure the yaml file required for Windows event logs a
 
 Any custom fields can be tagged along with the data in this step.
 
-Once the details are filled in, click on the **Download YAML File** button to get the yaml file.
+#### Enable process metric collection (Optional)
 
-<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Windows-OpenTelemetry/Windows-YAML.png' alt="YAML" />
+By default, the collector will not be sending process metrics to Sumo Logic this is because the number of processes running on a host can be very large, this may result in significant increase in Data Points per Minute (DPM).
```

Comment:
```suggestion
By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
```